src/service.js

```diff
@@ -0,0 +1,316 @@
+if(!global._babelPolyfill) { require('babel-polyfill'); }
+
+import Proto from 'uberproto';
+import mongoose from 'mongoose';
+import filter from 'feathers-query-filters';
+import errors from 'feathers-errors';
+import errorHandler from './error-handler';
+
+// Use native promises
+mongoose.Promise = global.Promise;
```

Comment:
@jamesjnadeau You're right but the babel polyfill already has native promises covered. The tests are all passing all the way back to v0.10 so we are good there.

Thanks for weighing in! It's nice to have some fresh eyes on this stuff.